test/sequential/test-tick-processor-preprocess-flag.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+require('../common');
+// const { isCPPSymbolsNotMapped } = require('./util');
```

Comment:
Accidentally left the commented out code?

Reply:
No, I wanted to check on what platforms it currently really fails and on which not (see my comment above in the description).